examples/text-classification/Cargo.toml

```diff
@@ -16,7 +16,9 @@ ndarray-blas-openblas = ["burn/ndarray", "burn/openblas"]
 tch-cpu = ["burn/tch"]
 tch-gpu = ["burn/tch"]
 wgpu = ["burn/wgpu"]
+wgpu-spirv = ["burn/wgpu-spirv"]
```

Comment:
Simply add `wgpu-spirv = ["wgpu", "burn/wgpu-spirv"]` so that you don't have to `#[cfg(any(feature = "wgpu", feature = "wgpu-spirv"))]` in the code.

Reply:
👍🏻